os/drivers/audio/alc1019.c

```diff
@@ -44,7 +44,9 @@
 #include "alc1019.h"
 #include "alc1019scripts.h"
 
-#define ALC1019_I2S_TIMEOUT_MS 200
+#define ALC1019_MIN_I2S_TIMEOUT_MS  10
+#define BYTE_TO_BIT_FACTOR          8
+#define SEC_TO_MSEC_FACTOR	    1000
```

Comment:
Can we move these declarations to some common header file?
MIN_I2S_TIMEOUT_MS must be common for all I2S?
And the other two declarations are very common. I think its already present somewhere in some header files.

Reply:
MIN_I2S_TIMEOUT_MS is a configurable parameter , but we can keep it common for all I2S . I will add these in header file